src/olympia/files/file_viewer.py

```diff
@@ -26,13 +22,23 @@
 from olympia.files.utils import (
     atomic_lock, extract_xpi, get_all_files, get_sha256)
 
-
-# Allow files with a shebang through.
-denied_magic_numbers = [b for b in list(blacklisted_magic_numbers)
-                        if b != (0x23, 0x21)]
-denied_extensions = [b for b in list(blacklisted_extensions) if b != 'sh']
 task_log = olympia.core.logger.getLogger('z.task')
 
+# Detect blacklisted files based on their extension.
```

Comment:
you'll make the ghost of andym cry with that comment.

Reply:
Indeed, fixed in 321dcb9e43